tox.ini

```diff
@@ -0,0 +1,19 @@
+[tox]
+envlist = unittest,py35-{pypi, github}
+
+
+[testenv:unittest]
+deps = -rrequirements_devel.txt
+commands = {envpython} -m pytest --cov=colcon_ros --cov-branch
+
+[testenv:py35-pypi]
+deps =
+    colcon-core
+    colcon-ros-bundle
+commands = ./run_integration_test.sh
+
+[testenv:py35-github]
+deps =
+    -e git+https://github.com/colcon/colcon-core.git#egg=colcon-core
```

Comment:
Should be installing any colcon deps of this package from master also. I think `colcon-cmake` is one.

Reply:
I added the other colcon dependencies being used from github master branch